wordlistctl.py

```diff
@@ -245,224 +242,235 @@ def lst_func(parser: argparse.ArgumentParser) -> None:
-			else:
-				argument["right"] = "No description available"
-			lmaxlen = max(lmaxlen, len(argument["left"]))
-			rmaxlen = max(rmaxlen, len(argument["right"]))
-
-		lwidth = lmaxlen
-		rwidth = max(0, self.width - lwidth - 4)
-		if (lwidth > int(self.width / 2) - 4):
-			lwidth = max(0, int(self.width / 2) - 4)
-			rwidth = int(self.width / 2)
-		outtmp = "  %-" + str(lwidth) + "s  %s"
-
-		lwrapper = textwrap.TextWrapper(width=lwidth)
-		rwrapper = textwrap.TextWrapper(width=rwidth)
-		for argument in self.positionals + self.options:
-			argument["left"] = lwrapper.wrap(argument["left"])
-			argument["right"] = rwrapper.wrap(argument["right"])
-
-
-		if (len(self.positionals) > 0):
-			output.append("")
-			output.append("Positionals:")
-			for positional in self.positionals:
-				for i in range(0, max(len(positional["left"]), len(positional["right"]))):
-					left = positional["left"][i] if (i < len(positional["left"])) else ""
-					right = positional["right"][i] if (i < len(positional["right"])) else ""
-					output.append(outtmp % (left, right))
-
-
-		if (len(self.options) > 0):
-			output.append("")
-			output.append("Options:")
-			for option in self.options:
-				for i in range(0, max(len(option["left"]), len(option["right"]))):
-					left = option["left"][i] if (i < len(option["left"])) else ""
-					right = option["right"][i] if (i < len(option["right"])) else ""
-					output.append(outtmp % (left, right))
-
-		if ("epilog" in self.program and self.program["epilog"] != "" and not str.isspace(self.program["epilog"])):
-			output.append("")
-			output.append(dewrapper.fill(self.program["epilog"]))
-
-		# Return output as single string
-		return str.join("\n", output)
-
-	
-	def print_usage(self, file=None):
-		if (file == None):
-			file = sys.stdout
-		file.write(self.format_usage() + "\n")
-		file.flush()
-
-	
-	def print_help(self, file=None):
-		if (file == None):
-			file = sys.stdout
-		file.write(self.format_help() + "\n")
-		file.flush()
-
-	def error(self, message):
-		sys.stderr.write(self.format_usage() + "\n")
-		sys.stderr.write(("Error: %s" % message) + "\n")
-		sys.exit(2)
+    def __init__(self, *args, width=78,
+                 **kwargs):  # this is basically just to get the variables for the screen, a bit like writing characters to the screen when youre making a kernel, place a character and then increase the y coordinates by x pixles
+        self.program = {key: kwargs[key] for key in kwargs}
+        self.positionals = []
+        self.options = []
+        self.width = width
+        super(ArgumentParser, self).__init__(*args, **kwargs)
+
+    def add_argument(self, *args, **kwargs):
+        super(ArgumentParser, self).add_argument(*args, **kwargs)
+        argument = {key: kwargs[key] for key in kwargs}
+
+        if (len(args) == 0 or (len(args) == 1 and isinstance(args[0], str) and not args[0].startswith("-"))):
```

Comment:
Drop all brackets for if. 

![image](https://user-images.githubusercontent.com/9546091/110033820-e3137380-7d5f-11eb-9eed-9a3dbf338a04.png)

Reply:
done